src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArr = sourceString
+    .split(';')
+    .map((element) => element.trim())
+    .filter((element) => element !== '');
+
+  const diferentArr = sourceArr.map((style) => style.split(':'));
+  const result = {};
+
+  if (!sourceString) {
+    return {};
+  }
+
+  diferentArr.map((value) => {
```

Comment:
Instead of using `map` here, which is intended for transforming arrays and returns a new array, consider using `forEach` since you are not using the returned array. This aligns with the checklist item: 'Avoid using loops in this task, use iteration methods instead.'